cromwell.example.backends/cromwell.examples.conf

```diff
@@ -157,6 +157,9 @@ call-caching {
   #   # The call caching blacklist cache is off by default. This cache is used to blacklist cache hit paths based on the
   #   # prefixes of cache hit paths that Cromwell has previously failed to copy for permissions reasons.
   #   enabled: true
+  #   # A blacklist grouping can be specified in workflow options which will inform the blacklister which workflows
+  #   # should share a blacklist cache.
+  #   grouping-workflow-option: call-cache-blackist-group
```

Comment:
Could this reuse the same config value as the hog safety group? We can rename it (or alias it elsewhere in the config) if "hog group" is unappetizing?

I think having multiple independent-but-actually-the-same versions of this value could end up getting more confusing than valuable?

Reply:
making this a first class concept is an interesting idea...